src/Sarif.Converters/ContrastSecurityConverter.cs

```diff
@@ -253,10 +256,11 @@ private Result ConstructAntiCachingControlsMissingResult(IDictionary<string, str
 
             string pageCount = locations.Count.ToString();
 
-            var result = new Result
+            const string RuleId = ContrastSecurityRuleIds.AntiCachingControlsMissing;
```

Comment:
>RuleId [](start = 25, length = 6)

DRY out the id: assign it once and use it twice rather than repeating the long name.